src/components/Profile/components/PersonalInfoList/index.js

```diff
@@ -285,10 +285,10 @@ const PersonalInfoList = ({
             <Grid container item xs={5} alignItems="center">
               <Typography>{'Mailbox:'}</Typography>
             </Grid>
-            <Grid container item xs={myProf ? 2 : 7} alignItems="center">
+            <Grid container item xs={myProf && mailCombo ? 2 : 7} alignItems="center">
               <Typography className={null}>{`#${Mail_Location}`}</Typography>
             </Grid>
```

Comment:
We want to show their Mail Location, even if they don't have a mailbox combo, right?

Reply:
Correct. This does that, it just changes the sizing based on whether mailCombo has value or not, because the list component has a certain structure.

Reply:
No worries! You scared me for a second bc initially I wrote the code that would've done the wrong thing you were thinking of and I thought maybe I accidentally committed it 😅